WebHostLib/static/assets/tutorial/ff1/multiworld_en.md

```diff
@@ -46,69 +39,30 @@ you prefer, or it is your first time playing select the "Archipelago" preset on
 It should download two files. One is the `*.nes` file which your emulator will run and the other is the yaml file
 required by Archipelago.gg
 
-### Generating the Multiworld and Starting the Server
-The game can be generated locally or by Archipelago.gg.
-
-#### Generating on Archipelago.gg (Recommended)
-1. Gather all yaml files
-2. Create a zip file containing all of the yaml files. Make sure it is a `*.zip` not a `*.7z` or a `*.rar`
-3. Navigate to the [Generate Page](https://archipelago.gg/generate) and click `Upload File`
-   1. For your first game keep `Forfeit Permission` as `Automatic on goal completion`. Forfeiting actually means
-      giving out all of the items remaining in your game in this case so you do not block anyone else.
-   2. For your first game keep `Hint Cost` at 10%
-4. Select your zip file
-
-#### Generating Locally
-1. Navigate to your Archipelago install directory
-2. Empty the `Players` directory then fill it with one yaml per player including your own which you got from the 
-   finalfantasyrandomizer website above
-3. Run `ArchipelagoGenerate.exe` (double-click it in File Explorer)
-4. You will find your generated game in the `output` directory
-
-#### Starting the server
-If you generated on Archipelago.gg click `Create New Room` on the results page to start your server
-If you generated locally simply navigate to the [Host Game Page](https://archipelago.gg/uploads) and upload the file
-in the `output` directory
-
-### Running the Client Program and Connecting to the Server
+## Running the Client Program and Connecting to the Server
+
```

Comment:
To connect to the previous statement above I would just add a quick note here

`Once the game has been hosted`